### ISSUE TITLE
CI Builds fail due to missing Git tag

### ISSUE THREAD

**Issue by offa** [offa/conan-restclient-cpp#6]
All current CI builds have failed with following error:
```
[…]
conans.errors.CalledProcessErrorWithStderr: Command 'git -c http.sslVerify=false checkout "0.5.1"' returned non-zero exit status 1.
error: pathspec '0.5.1' did not match any file(s) known to git.
[…]
```

**Comment by offa** [comment]
Local build still works fine.

**Comment by offa** [comment]
Upstream fix is in progress :+1:

**Comment by offa** [comment]
Fixed with Conan 1.8.1.
